.github/workflows/build.yml

```diff
@@ -230,3 +230,28 @@ jobs:
           rustup component add rustfmt
       - name: Run rustfmt checks
         run: ci/rustfmt.sh
+
+  incremental-mutants:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Relative diff
+        run: |
+          git branch -av
+          git diff origin/main.. | tee git.diff
+      - uses: Swatinem/rust-cache@v2
+      - uses: taiki-e/install-action@v2
+        name: Install cargo-mutants using install-action
+        with:
+          tool: cargo-mutants
+      - name: Mutants
+        run: |
+          cargo mutants --no-shuffle -j 2 -vV --in-diff git.diff
+      - name: Archive mutants.out
```

Comment:
I don't believe we'll need this either. It'd only be useful if we had some perhaps custom app to feed it to so it could leave a PR comment like codecov. Will remove for now.